examples/aws-microk8s/cloud-formation.yaml

```diff
@@ -92,24 +112,36 @@ Resources:
     Properties:
       LaunchTemplateName: !Sub "${AppName}"
       LaunchTemplateData:
-        UserData: "IyEvYmluL2Jhc2gKc25hcCBpbnN0YWxsICBqb3VsZS1leHBhbnNpb24gLS1jbGFzc2ljCnNuYXAgaW5zdGFsbCBtaWNyb2s4cyAtLWNsYXNzaWMgLS1jaGFubmVsIDEuMjEvc3RhYmxlCnVzZXJtb2QgLWEgLUcgbWljcm9rOHMgdWJ1bnR1Cm1rZGlyIC9ob21lL3VidW50dS8ua3ViZQpzdWRvIGNob3duIC1mIC1SIHVidW50dSAvaG9tZS91YnVudHUvLmt1YmUKc25hcCBzZXQgam91bGUtZXhwYW5zaW9uIHByb3ZpZGVyPWF3cwpzbmFwIHNldCBqb3VsZS1leHBhbnNpb24gYXBwbGljYXRpb25zPW1pY3Jvazhz"
+        UserData:
+          Fn::Base64:
+            Fn::Join:
```

Comment:
I think it is better to use `!Sub` for this, similar to what is shown in the YAML section [in the AWS docs for Fn::Sub](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html)

For example, something like:

```yaml
        UserData:
          Fn::Base64:
            !Sub |
              #!/bin/bash

              snap install joule-expansion --classic
              snap install microk8s --classic --channel=${KubernetesVersion}/stable
              usermod -a -G microk8s ubuntu
              mkdir /home/ubuntu/.kube
              chown -f -R ubuntu /home/ubuntu/.kube
              snap set joule-expansion provider=aws
              snap set joule-expansion application=microk8s
```